app/assets/javascripts/lib/models/input_element.coffee

```diff
@@ -27,6 +28,10 @@ class @InputElement extends Backbone.Model
 
     conversions
 
+  googleAnalytics: =>
+    if ga && ga.loaded
+      ga('send', 'event', 'slider-change', "#{@get 'key'}")
```

Comment:
Nitpicking: Can this be made `input-change` instead of `slider-change`? ETModel tends to use the term "input" or "input element" as not all inputs are sliders (merit order switch, FCE switch, etc).